src/community/mbstyle/pom.xml

```diff
@@ -62,6 +62,24 @@
 	      <artifactId>gs-restconfig</artifactId>
 	      <version>${project.version}</version>
 	      <scope>test</scope>
-	    </dependency>        
+	    </dependency>
+  	    <dependency>
+          <groupId>org.geoserver.wms</groupId>
```

Comment:
This dependency looks like a malformed duplicate duplicate of the other gs-web-wms dependency below (The correct groupId is `org.geoserver.web`). Try removing this one and see if everything still works.